packages/cardpay-sdk/sdk/reward-manager/base.ts

```diff
@@ -35,6 +44,159 @@ export default class RewardManager {
     return await (await this.getRewardManager()).methods.getRewardSafeOwner(rewardSafeAddress).call();
   }
 
+  async withdraw(txnHash: string): Promise<TransactionReceipt>;
+  async withdraw(
+    safeAddress: string,
+    to: string,
```

Comment:
On first glance, this looks good, the one comment I have is that the "to" is only allowed to be the owner of the reward safe at the moment. So we could calculate it, but maybe it still makes sense to pass it in in case we allow other addresses in future

Reply:
So @alex-cardstack this is actually oversight by me when I reviewed the pr on card protocol. Reason why I put `to` is because the `withdraw` should go to ANY address not just the owner's address. We can set a ticket to fix this in the future.